pwsh/Helpers.ps1

```diff
@@ -0,0 +1,138 @@
+<#
+.SYNOPSIS
+  This function is used to tag a commit and create a release on GitHub.
+#>
+function New-GitHubRelease {
+  param(
+    # An API token that provides authentication to GitHub
+    [string]$AuthToken,
+    # The SHA of the commit to create the tag and release on
+    [string]$GitSHA,
+    # The decscription of the release - usually release notes
+    [string]$ReleaseBody,
+    # The desired name for the release
+    [string]$ReleaseName,
+    # The name of the repository to create the release on
+    [string]$RepoName,
+    # The owner (username/organisation) of the repository to create the release on
+    [string]$RepoOwner,
+    # The name to give the created tag
+    [string]$TagName,
+    # Mark this release as a Draft release - do not publish it
+    [bool]$Draft,
+    # Mark this release as a pre-release
+    [bool]$PreRelease
+  )
+
+  $releases_url = "https://api.github.com/repos/$RepoOwner/$RepoName/releases"
```

Comment:
this release URL is recreated in `Publish-ReleaseAsset` below. We should only do this one; the `{ReleaseId}/assets` piece can be added by the publish assent code